testsuite/tests/typing-warnings/coercions.compilers.principal.reference

```diff
@@ -1,26 +1,25 @@
-
-#     Characters 76-79:
+Characters 168-171:
```

Comment:
(location change; I'm not saying this must be fixed, just that I'm not sure where it comes from. Maybe the first sentence has its location offset by the test comment header?)